lnwallet/wallet.go

```diff
@@ -32,13 +33,23 @@ const (
 	// The size of the buffered queue of requests to the wallet from the
 	// outside word.
 	msgBufferSize = 100
+
+	// anchorChanReservedValue is the amount we'll keep around in the
+	// wallet in case we have to fee bump anchor channels on force close.
+	anchorChanReservedValue = btcutil.Amount(1000)
```

Comment:
Seems too minimal to actually have an impact if we do indeed need to expedite confirmation due to expiring time-locks, etc.

Reply:
Bumped to `10_000` for now, still trying to figure out if there is a "better" constant or if we should try to make somehting dynamioc.